package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "rdflib",
   "description": "an RDF library for node.js. Suitable for client and server side.",
-  "version": "0.15.0",
+  "version": "0.14.0",
```

Comment:
This looks like a mistake in your rebase. You'll want to leave the current version at 0.15.0.